src/workloads/execution/TimeSeries2dsphere.yml

```diff
@@ -0,0 +1,145 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This test exercises the behavior of querying for data in a timeseries collection using a 2dsphere index.
+
+  The phases are:
+  1. Create collection
+  2. Insert events + create index
+  3. Quiesce
+  4. Query
+
+GlobalDefaults:
+  # Schema constants.
+  dbname: &db test
+  fieldName: &field "numeric"
+  index: &index
+    keys: {location: '2dsphere'}
+
+  # Data distribution constants.
+  # To insert more efficiently, pick a big-ish batch size.
+  # This shouldn't really affect the resulting collection though.
+  # TODO just inline these if they're used once.
+  batchSize: &batchSize 30000
+  # With events 10 seconds apart, a 
+  docsPerSeries: &docsPerSeries 1000
+  documentCount: &documentCount 1036800
+  # Since we have a relatively small amount of data, we don't want to use an area that's too
+  # big (such as the entire Earth, or a whole continent). That would make the data very
+  # sparse over that area. These values roughly correspond to New York State, which is
+  # hopefully a good scale.
+  minX: &minX -80.0
+  maxX: &maxX -73.0
+  minY: &minY 40.0
+  maxY: &maxY 45.0
+  # Suppose each sensor moves at around 60 mph or 100 kph.
+  # This comes out to around 30 m/s.
+  # Events are 10 seconds apart, so that's 300m per event.
+  # 90 degrees of longitude or latitude is roughly 10,000 km = 10,000,000m.
+  # So (300m / event) * (90 deg / 10,000,000m) = 0.0027 deg / event.
+  # This is very rough, and doesn't account for how 1 degree longitude has a different length
+  # at different latitudes. The idea is just to get roughly the right scale.
+  # Another check: 0.0027 degrees per doc and 1,000 docs per series means 2.7 degrees per
+  # series, which is roughly the scale of our min/max X/Y.
+  distPerDoc: &distPerDoc 0.0027
+
+  # Other abbreviations.
+  nop: &Nop {Nop: true}
+
+
+Actors:
+- Name: CreateTimeSeriesCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: *db
+    Operation:
+      OperationMetricsName: CreateTimeSeriesCollection
+      OperationName: RunCommand
+      OperationCommand:
+        {create: &coll Collection1, timeseries: {timeField: "timestamp", metaField: "meta"}}
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    # Collection: *coll
+    AreYOuEvenListening: false
```

Comment:
```suggestion
```

Reply:
Oops, thanks. 😀  I think I was confused about the behavior of Loader--it must be ignoring that `Collection:` argument, because it ignores other unexpected arguments.